src/main.cpp

```diff
@@ -2235,6 +2235,9 @@ bool UndoReadFromDisk(CBlockUndo& blockundo, const CDiskBlockPos& pos, const uin
     CHashWriter hasher(SER_GETHASH, PROTOCOL_VERSION);
     hasher << hashBlock;
     hasher << blockundo;
+
+    LogPrint("sc", "%s\n", blockundo.ToString());
+
```

Comment:
How big is this blockundo? I wonder if this log does affect prod performances in any way

Reply:
We are printing the log if the category is explicitly set in the -debug option list